src/stepFunctions/activation.ts

```diff
@@ -40,11 +40,11 @@ async function registerStepFunctionCommands(
     const visualizationManager = new AslVisualizationManager(extensionContext)
 
     extensionContext.subscriptions.push(
-        vscode.commands.registerCommand('aws.previewStateMachine', async () => {
+        vscode.commands.registerCommand('aws.previewStateMachine', async (textEditor?: vscode.TextEditor) => {
             try {
                 return await visualizationManager.visualizeStateMachine(
                     extensionContext.globalState,
-                    vscode.window.activeTextEditor
+                    textEditor || vscode.window.activeTextEditor
                 )
             } finally {
                 telemetry.recordStepfunctionsPreviewstatemachine()
```

Comment:
Unrelated to this PR, but nit: should this reflect the success/failure of the above call?

Reply:
Yes, seems like it. Let me think about it. I'm adding a TODO comment for now.